internal/desktop/app.go

```
@@ -0,0 +1,962 @@
+	a.auth = authStatus{State: "error", Error: err.Error()}
+	a.authMu.Unlock()
+}
+
+func (a *App) getClient() (gmailclient.Client, error) {
+	a.clientMu.Lock()
+	defer a.clientMu.Unlock()
+	if a.client != nil {
+		return a.client, nil
+	}
+	if a.cfg.Client == nil {
+		return nil, errors.New("Gmail client is not configured")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)